util/judge.py

```diff
@@ -0,0 +1,188 @@
+import os
+import sys
+from datetime import datetime as dt
+from datetime import timedelta
+from typing import List
+
+import ccxt
+import requests
+from enforce_typing import enforce_types
+from ocean_lib.models.data_nft import DataNFT
```

Comment:
df-py already has its way of accessing ocean with a "thin wrapper" of the contracts, via `util/oceanutil.py`. By design, it avoids importing ocean.py because of the large additional complexity.

Please leverage it here. Avoid adding in ocean.py.

Reply:
ok! ocean.py is a dependency of predict-eth-helpers, though. I will also have to copy the helpers I'm using.

Reply:
Yes, do copy. We don't want that dependency either.